library/mono

```diff
@@ -83,13 +83,23 @@
 4.6.0-onbuild: git://github.com/mono/docker@9fdd0e79b4eb3e7e7e818fbd58bd324d4c5ab7e1 4.6.0.245/onbuild
 
 4.6.1.3: git://github.com/mono/docker@f0f1e0b9f693bd7c58d9623d402c8dc69234bbe1 4.6.1.3
```

Comment:
Since this version and its onbuild is not going to get updates we'd rather drop it from the list.  It will still be available to pull from the docker hub, but will no longer appear under the "Supported Versions".